docs/migration/5_0.md

```diff
@@ -1,5 +1,8 @@
 # Technical Changes in Spartacus 5.0
 
+Spartacus migration schematics tries to find in your codebase usages of Spartacus' public API (e.g. functions/methods/classes) which were changed (in a breaking way) or removed in version 5.0. The migration schematics will insert a code comment with a hint "TODO:Spartacus" above such lines of code, in order to assist you with the migration. 
+Note: Please beware that if by coincidence you have in your codebase a usage of a method/function/class that is named the same as the changes or removed Spartacus' public APIs, there might be a chance that it may be incorrectly classified as Spartacus' API and the migration schematics might insert a comment there as well (even though it shouldn't). In that case, please ignore and remove such a comment.
```

Comment:
```suggestion
Note: Please beware that if by coincidence you have in your codebase a usage of a method/function/class that is named the same as the changed or removed Spartacus' public APIs, there might be a chance that it may be incorrectly classified as Spartacus' API and the migration schematics might insert a comment there as well (even though it shouldn't). In that case, please ignore and remove such a comment.
```